views/templates/admin/tabs/content/config.tpl

```diff
@@ -51,11 +71,11 @@
         </form>
     {/foreach}
     <div class="panel-footer">
-        <div class="col-lg-10 col-xs-10">
-            <button name="refreshPage" type="submit"
-                    class="btn btn-secondary refreshPage">{l s='Return' mod='psreassurance'}</button>
+        <div class="col-xs-6 text-left">
+            <input name="refreshPage" type="submit"
+                    class="btn btn-primary refreshPage" value="{l s='Return' mod='psreassurance'}">
```

Comment:
I wonder since the module class uses `Modules.Blockreassurance.Admin` as a translation domain, shouldn't it be the same thing for templates? @LouiseBonnard what do you think?

Reply:
Yep, we must stay consistent and keep on using _Modules.Blockreassurance.Admin_ as translation domain. But I can't find this wording within the module, is it because it is a template?

Reply:
Updated